src/zc/zodbdgc/__init__.py

```diff
@@ -603,12 +613,12 @@ def check_(config, references=None):
                 if not seen.insert(name, oid):
                     continue
                 p, tid = storages[name].load(oid, b'')
-                if (# XXX should be in is_blob_record
+                if (  # XXX should be in is_blob_record
                     len(p) < 100 and (b'ZODB.blob' in p)
                         and ZODB.blob.is_blob_record(p)
                 ):
                     storages[name].loadBlob(oid, tid)
-            except:
+            except BaseException:
```

Comment:
I personally dislike this change because `except BaseException` is just a longer way of saying exactly the same thing as `except:`. It doesn't actually solve the problem of catching too much, it just makes it *look* that way at a glance. I'd be happier with a pragma making an exception for lines like this (`# pylint:disable=broad-except`).

In this particular case, I suspect `except Exception:` is actually probably more than sufficient, and `except POSError:` is probably what it should really be.